travis_script.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+sudo apt-get -qq update
+sudo apt-get install ninja-build -y
+sudo apt-get install zlib1g-dev -y
```

Comment:
You could install packages in `.travis.yml` then this would really only the test setup. In any case, putting both packages in one `apt-get install` safes some space :)